### PR TITLE
initial release

### PR DESCRIPTION
**Pull request by Godley**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+before_script:
+- docker pull "$IMAGE_NAME" || true
+script:
+- docker build --pull --tag "$IMAGE_NAME" .
+after_script:
+- docker images
+before_deploy:
+- docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
+- docker tag "$IMAGE_NAME" "${IMAGE_NAME}:latest"
+- docker tag "$IMAGE_NAME" "${IMAGE_NAME}:${TRAVIS_COMMIT}"
```
Comment (mikebryant) [inline .travis.yml]: Should we do this only on tags? Use semantic versioning?

(And possibly worth not even making available `:latest`?)

Comment (stuart-warren) [reply .travis.yml]: I'd agree with on tags and no latest.

Might want to add metadata labels like we do in docker-build

Comment (Godley) [reply .travis.yml]: `latest` makes the readme/running instructions a little bit more complicated

Comment (Godley) [reply .travis.yml]: there's no explicit reference to `tag` in the `present` section of this but think I've done it right: https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#Conditional-Builds

**Pull request merged**